sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLQueryPlanner.java

```diff
@@ -129,11 +129,16 @@ private FrameworkConfig defaultConfig(JdbcConnection connection, RuleSet[] ruleS
     final SqlOperatorTable opTab0 =
         connection.config().fun(SqlOperatorTable.class, SqlStdOperatorTable.instance());
 
+    Object[] contexts =
+        org.apache.beam.vendor.calcite.v1_20_0.com.google.common.collect.ImmutableList.of(
+                connection.config(), TableResolutionContext.joinCompoundIds("datacatalog"))
```

Comment:
This should not be necessary. The pipeline registers "datacatalog" as the default table provider.

Reply:
This line is not to register default table provider. Instead, it's used to load table from "datacatalog". Basically this is a quick fix so I won't expect we have a sold implementation at this moment.

Reply:
And this line.